R/dependencies.R

```diff
@@ -631,3 +618,62 @@ fileDependencies.Rmd.evaluate <- function(file) {
 
   unique(unlist(deps, recursive = TRUE))
 }
+
+
+
+
+
+fileDependencies.Rmd.tangle <- function(file, encoding = "UTF-8") {
+
+  # discovered packages
+  deps <- list()
+
+  # unique key (line) to split R code with
+  key <- paste0("###--packrat", as.integer(Sys.time()), "\n")
```

Comment:
The presence of `Sys.time()` suggests that time matters, but it doesn't actually matter in this case. Using something like `sample(letters, 10, replace = TRUE)` also creates a unique-ish key but doesn't suggest that time matters, so it would be a little clearer.